app/views/notifications/_list.html.erb

```diff
@@ -23,11 +23,13 @@
             <%= I18n.l notification.created_at,format: :short%> (<%= time_ago_in_words notification.created_at %> ago)
           </span>
         </h5>
-        <%= simple_format(notification.message).html_safe %>
+        <div class="<%= 'text-danger' if notification.error? %>">
```

Comment:
red text on the light blue background might look a little strange
![image](https://user-images.githubusercontent.com/1652117/55312578-296e4280-5466-11e9-8f5f-3a99da7c2a5a.png)
Maybe we can use a different background color to indicate the nature of the notification as an error? Or add a small FontAwesome icon?

Reply:
I don't want to add special css for a new background color (bootstrap does not support light backgrounds by default, the danger-background is too strong).
I will look into using an icon then

Reply:
![image](https://user-images.githubusercontent.com/1346493/55360232-0b3f2b80-54d4-11e9-86aa-f968743c9234.png)
Using an icon looks fine to me.

Reply:
:+1: